### ISSUE TITLE
Document requirements and Implement model run(s) for ADCIRC (ADvanced CIRCulation) 

### ISSUE THREAD

**Issue by ZacharyWills** [ioos/Cloud-Sandbox#58]
# Welcome to the IOOS Cloud-Sandbox! 
## Expected Behavior

A ADCIRC (ADvanced CIRCulation) model version should be selected, compiled and made available to users of the Cloud-Sandbox.

## Current Behavior

ADCIRC (ADvanced CIRCulation)  is not available.

## Possible Solution

The Source should be compiled periodically, a set of static configuration examples made available, and documentation should be written to describe not only how this is done but also the lifecycle of this model.

## Steps to Reproduce or Implement:

- [ ] ADCIRC source code acquired
- [ ] Compilation and test-runs automated for availability
- [ ] Available Data acquired, and it's location documented for community use 
- [ ] Documentation of full ADCIRC usage, necessary inputs, output location, etc. made available

## Context

The National Ocean Service has adopted four different community code packages for use in the operational Coastal
Ocean Modeling Framework (COMF): 

## Your Environment

* Version used: To be updated upon Selection
* Software Name and version: ADCIRC (ADvanced CIRCulation) model
* Operating System and version (desktop or mobile): Amazon Linux 2
* Link to your project/sponsor: tiffany.c.vance@noaa.gov

**Comment by janahaddad** [comment]
Hello, I'm getting familiar with the sandbox and was trying to compile ADCIRC using cmake in the NOAA EC2 instance (for eventually testing ADCIRC+SWAN model runs for CO-OPS CORA)

Cmake is not finding a fortran compiler on the system, and I'm wondering if I'm going about this the wrong way. Should I be cloning cloud-sandbox repo first? or can you provide a path for a fortran compiler? or should I install gfortran in my user directory? 

![image](https://github.com/ioos/Cloud-Sandbox/assets/18744882/ed3b76a0-ea25-4296-bc03-7c512e7a96bc)

Here's the full output from the Cmake GUI configuration:
![image](https://github.com/ioos/Cloud-Sandbox/assets/18744882/b62a9a68-7617-45b2-8169-d601b4c758ea)

thanks for your help!  

cc @Michael-Lalime, @greg-dusek

**Comment by Michael-Lalime** [comment]
@ZacharyWills @KatherinePowell-NOAA @patrick-tripp I thought we had a Fortran compiler in the sandbox? Can any of you help with Jana's problem with cmake and Fortran?

**Comment by patrick-tripp** [comment]
Hi Jana,

You need to load the appropriate modeles that contain the libraries and compilers you want to use.
'spack load' might also work also.
'module avail'
'spack find'

**Comment by ZacharyWills** [comment]
Here's the fortran install/version:
https://github.com/ioos/Cloud-Sandbox/blob/7ecc756d85b9b9ea92be0e062331738536e45649/scripts/funcs-setup-instance.sh#L379

**Comment by patrick-tripp** [comment]
The above is for an older version and should be removed. When the sandbox is deployed it will automatically setup all of the software and libraries you need using the setup-instance.sh script. If it didn't run, you can manually run the scripts/setup-instance.sh script which will use Spack to install everything you need with the proper dependencies.

**Comment by Michael-Lalime** [comment]
I think Jana is using the currently deployed version of the sandbox and not
deploying a new instance.

**Comment by janahaddad** [comment]
> I am assuming you ran the setup-instance script when your sandbox was deployed?
> That will install everything you need.

@patrick-tripp No, where is setup-instance? I believe that I'm working in an environment that already has the latest cloud-sandbox deployed, right @Michael-Lalime?

**Comment by patrick-tripp** [comment]
OK, you need to specify the modules you need to use in your build and run scripts. You can see the LiveOcean or nosofs models as a reference.

**Comment by Michael-Lalime** [comment]
@Jana Haddad - NOAA Affiliate ***@***.***> Yes you are using the
version that was deployed in December. There might be updates since then
that haven't been incorporated but Patrick's suggestions should help you
get going.

**Comment by KatherinePowell-NOAA** [comment]
Lynker Coastal Modelers (JS, Julio, and Jason) completed documentation of ADCIRC model requirements and tests in the [Lynker Google Drive, Technical docs folder](https://drive.google.com/drive/folders/1D8bxz26dJSPmXEId7wXi_e22f9OCcjAf?usp=drive_link).
